specs/DotNetProjectFile.Analyzers.Specs/Design_specs.cs

```diff
@@ -1,3 +1,5 @@
+#pragma warning disable RS1035 // Do not use APIs banned for analyzers
```

Comment:
Perhaps we should just disable this on a project level in a seperate PR, because this is the zillionth time I have done or seen this in the test project (also our analyzer does quite some IO, so perhaps on a solution level).

Reply:
See #355